mir_eval/transcription.py

```diff
@@ -0,0 +1,243 @@
+'''
+The aim of a transcription algorithm is to produce a symbolic representation of
+a recorded piece of music in the form of a set of discrete notes. There are
+different ways to represent notes symbolically. Here we use the piano-roll
+convention, meaning each notes has a start time, a duration (or end time), and
+a single, constant, pitch value. Pitch values can be quantized (e.g. to a
+semitone grid tuned to 440 Hz), but do not have to be. Also, the transcription
+can contain the notes of a single instrument or voice (for example the melody),
+or the notes of all instruments/voices in the recording. This module is
+instrument agnostic: all notes in the estimate are compared against all notes
+in the reference.
+
+There are many metrics for evaluating transcription algorithms. Here we limit
+ourselves to the most simple and commonly used: given two sets of notes, we
+count how many estimate notes match the reference, and how many do not. Based
+on these counts we compute the precision, recall, and f-measure of the estimate
+given the reference. The default criteria for considering two notes to be a
+match are adopted from the MIREX "Multiple fundamental frequency estimation and
+tracking, task 2" criteria:
+(http://www.music-ir.org/mirex/wiki/2015:Multiple_Fundamental_Frequency_\
+Estimation_%26_Tracking_Results_-_MIREX_Dataset#Task_2:Note_Tracking_.28NT.29):
```

Comment:
@craffel I'm not sure, but right now sphinx doesn't seem to include the new transcription code in the compiled docs at all. I guess I'm supposed to change something somewhere, or re-build something somewhere?